league/tactician-bundle/0.4/config/packages/league_tactician.yaml

```diff
@@ -0,0 +1,17 @@
+# Library documentation: http://tactician.thephpleague.com/
+# Bundle documentation: https://github.com/thephpleague/tactician-bundle/blob/v0.4.1/README.md
+tactician:
+    default_bus: default
+    commandbus:
+        default:
+            middleware:
+                - tactician.middleware.command_handler
+# Example of separate command/query buses with transactional command bus
+#    query:
```

Comment:
Indentation looks weird. Should be 4 more sapces to me.

Reply:
thanks, good catch!